client/app/components/packages/pas-form/edit.hbs

```diff
@@ -820,37 +845,37 @@
             </a>
           </li>
           <li class="medium-margin-bottom">
-            <a href="#">
+            <a href="#project-geography">
               <FaIcon @icon="chevron-circle-down" @fixedWidth={{true}} class="text-silver" />
               Project Geography
             </a>
           </li>
           <li class="medium-margin-bottom">
-            <a href="#">
+            <a href="#proposed-land-use-actions">
               <FaIcon @icon="chevron-circle-down" @fixedWidth={{true}} class="text-silver" />
               Proposed Land Use Actions
             </a>
           </li>
           <li class="medium-margin-bottom">
-            <a href="#">
+            <a href="#project-area">
               <FaIcon @icon="chevron-circle-down" @fixedWidth={{true}} class="text-silver" />
               Project Area
             </a>
           </li>
           <li class="medium-margin-bottom">
-            <a href="#">
+            <a href="#proposed-development-site">
               <FaIcon @icon="chevron-circle-down" @fixedWidth={{true}} class="text-silver" />
               Proposed Development Site
             </a>
           </li>
           <li class="medium-margin-bottom">
-            <a href="#">
+            <a href="#project-description">
               <FaIcon @icon="chevron-circle-down" @fixedWidth={{true}} class="text-silver" />
               Project Description
             </a>
           </li>
           <li class="medium-margin-bottom">
-            <a href="#">
+            <a href="#attached-documents">
```

Comment:
These still have the `#`

Reply:
The `href` are supposed to. The `id` are not.

Reply:
oooyyyyyyyy duh! thanks